evap/evaluation/migrations/0109_import_names.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import migrations, models
+
+
+def fill_import_names(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    Degree = apps.get_model('evaluation', 'Degree')
+    for degree in Degree.objects.using(db_alias).all():
```

Comment:
No, the `using(db_alias)` is not postgres-related. I think I added it because of the warning for [`RunPython`](https://docs.djangoproject.com/en/3.0/ref/migration-operations/#runpython), but I guess that most of the migrations don’t use that, so we should be fine without it…

To make a simple `update` work, I need an expression that creates an array based on a field. I tried `Degree.objects.update(import_names=[F('name_de')])`, but this sets all import_names to the string `name_de`, not the value of the field. Other quick option would be to write raw sql, but I think that is unnecessary here.

Reply:
Ah, never mind, I just messed up there in the first comment. I thought you were giving each degree the same, constant value, but each degree gets a different one.